spec/models/user_spec.rb

```diff
@@ -71,4 +71,16 @@
       expect(user.scoped_teams).to eq(Team.all)
     end
   end
+
+  describe 'validates' do
+    it 'uniqueness of email address' do
```

Comment:
can we add a case where we delete a user and then are able to create a new account with that same email still?